src/App.jsx

```
@@ -0,0 +1,178 @@
+import { useState, useEffect, useCallback } from 'react'
+import TodoList from './components/TodoList'
+import TodoForm from './components/TodoForm'
+import TodoFilter from './components/TodoFilter'
+import TodoStats from './components/TodoStats'
+import './App.css'
+
+function App() {
+  var testVariable = 123 // Renamed to camelCase for rule test
```

ESLint

[error] 'testVariable' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)